packages/transformers/image/src/ImageResizeTransformer.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import {Transformer} from '@parcel/plugin';
+import sharp from 'sharp';
+
+const FORMATS = new Map([
+  ['heic', 'heif'],
+  ['heif', 'heif'],
+  ['jpeg', 'jpeg'],
+  ['jpg', 'jpeg'],
+  ['png', 'png'],
+  ['raw', 'raw'],
+  ['tiff', 'tiff'],
+  ['webp', 'webp'],
+]);
+
+export default new Transformer({
+  async transform({asset}) {
+    asset.isIsolated = true;
+
+    let inputBuffer = await asset.getBuffer();
+    let width = asset.query.width ? parseInt(asset.query.width, 10) : null;
+    let height = asset.query.height ? parseInt(asset.query.height, 10) : null;
+    let format = asset.query.as ? asset.query.as.toLowerCase().trim() : null;
+
+    let imagePipeline = sharp(inputBuffer);
+    if (width || height) {
+      imagePipeline.resize(width, height);
+    }
+
+    if (format) {
+      if (!FORMATS.has(format)) {
+        throw new Error(`Sharp does not support ${format} images.`);
+      }
+
+      asset.type = format;
+
+      imagePipeline[FORMATS.get(format)]();
```

Comment:
Nearly all formats (except raw) have a common `quality` attribute in the "options" argument that's passed into the format methods. Perhaps `?quality={number}` would be a good candidate for query param, and could be passed in here as a parameter as `{ quality }`?

See: https://sharp.pixelplumbing.com/api-output

Reply:
Seems fair enough, will add it in later today